.travis.yml

```diff
@@ -1,33 +0,0 @@
-dist: xenial
-sudo: required
-services:
```

Comment:
why we are removing this file?

Reply:
We are not longer using it. This file and others (`.travis.yml`, `GNUmakefile`) were provided by HashiCorp when they were hosting our repo and building the provider (using TravisCI).